### DISCUSSION TITLE
Duplicate multiple events/lines?

### DISCUSSION THREAD

**Discussion by cordulius** [cpvalente/ontime#1312] [Q&A]
Hi,

I'm working on various projects where events/lines are repeated/need to be duplicated.
Sadly, I can only duplicate events 1-by-1 so far... Is there a way to select multiple events and duplicate them? If not, could that be implemented?

Thank you. :)

**Comment by alex-Arc** [comment]
Hey @cordulius 
We are dragging our feet here, waiting for some other stuff to be done first 
But we are hoping to make it (or something like it) possible in the future
It is just not the immediate next step
